packages/addons/addon-depends/pcre2/package.mk

```diff
@@ -0,0 +1,51 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="pcre2"
+PKG_VERSION="10.30"
+PKG_SHA256="90bd41c605d30e3745771eb81928d779f158081a51b2f314bbcc1f73de5773db"
+PKG_LICENSE="BSD"
+PKG_SITE="http://www.pcre.org/"
+PKG_URL="https://ftp.pcre.org/pub/pcre/$PKG_NAME-$PKG_VERSION.tar.bz2"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5. PCRE has its own native API, as well as a set of wrapper functions that correspond to the POSIX regular expression API. The PCRE library is free, even for building commercial software."
+PKG_USE_CMAKE="no"
+PKG_AUTORECONF="no"
```

Comment:
Small nit - after #2117, can you reverse the two lines above so that it's:
```
PKG_AUTORECONF="no"
PKG_USE_CMAKE="no"
```
Thanks!